nemo/collections/nlp/data/text_classification/text_classification_dataset.py

```diff
@@ -69,7 +69,7 @@ def __init__(
         tokenizer: Any,
         max_seq_length: int,
         num_samples: int = -1,
-        shuffle: bool = False,
+        shuffle: bool = False, # Q: Why shuffling is here? Shouldn't DataLoader take care of it?
```

Comment:
it is passed to data loader. We have all in one place so that users don't think about dataset Vs dataloader

Reply:
It has the same shuffle value as dataloaders. We may drop it. Is there any case in multi-node/gpu training where it matters if data are shuffled or not before passing to the data loaders?